search/tests/test_course_discovery_views.py

```diff
@@ -403,3 +405,238 @@ def test_combined_facet_filter_aggregated_correctly(self):
         self.assertIn("org", aggs)
         self.assertIn("OrgA", aggs["org"]["terms"])
         self.assertIn("OrgC", aggs["org"]["terms"])
+
+
+@override_settings(
+    SEARCH_ENGINE="search.tests.utils.ForceRefreshElasticSearchEngine",
+    COURSEWARE_CONTENT_INDEX_NAME=TEST_INDEX_NAME,
+    COURSEWARE_INFO_INDEX_NAME=TEST_INDEX_NAME,
+)
+class TestElasticsearchSingleValueDiscoveryUrl(TestCase, SearcherMixin):
+    """
+    Integration tests for Elasticsearch + /course_discovery/ endpoint
+    """
+    def setUp(self):
```

Comment:
Please add a blank line between the doctring and the method, and please add docstrings to the tests, describing what is being tested.